cassovary-core/src/test/scala/com/twitter/cassovary/graph/GraphBehaviours.scala

```diff
@@ -19,7 +19,7 @@ import org.scalatest.{Matchers, WordSpec}
 trait GraphBehaviours extends Matchers {
```

Comment:
just make this trait of type [V <: Node] and simply use DirectedGraph[V] in all methods below without specifying [V<; Node] in every method